public/include/definitions_global.php

```diff
@@ -176,19 +176,181 @@ class Definitions {
 		'B748' => 9
 	);
 
+	private static $generalAviationAircraft = array( //Source: http://www.flugzeuginfo.net/table_accodes_en.php
+  		'B18T', //Beech
+  		'B190',
+  		'B350',
+  		'B36T',
+  		'BE10',
+  		'BE17',
+  		'BE18',
+  		'BE20',
+  		'BE23',
+  		'BE24',
+  		'BE30',
+  		'BE35',
+  		'BE36',
+  		'BE40',
+  		'BE50',
+  		'BE55',
+  		'BE58',
+  		'BE60',
+  		'BE65',
+  		'BE70',
+  		'BE76',
+  		'BE77',
+  		'BE80',
+  		'BE88',
+  		'BE95',
+  		'BE99',
+  		'BE9L',
+  		'BE9T',
+  		'C02T', //Cessna
+  		'C04T',
+  		'C06T',
+  		'C07T',
+  		'C10T',
+  		'C120',
+  		'C140',
+  		'C150',
+  		'C152',
+  		'C162',
+  		'C170',
+  		'C172',  		
+  		'C175',
+  		'C177',
+  		'C180',
+  		'C182',
+  		'C185',
+  		'C188',
+  		'C190',
+  		'C195',
+  		'C205',
+  		'C206',
+  		'C207',
+  		'C208',
+  		'C210',
+  		'C21T',
+  		'C25A',
+  		'C25B',
+  		'C303',
+  		'C310',
+  		'C320',
+  		'C335',
+  		'C336',
+  		'C337',
+  		'C340',
+  		'C402',
+  		'C404',
+  		'C411',
+  		'C414',
+  		'C421',
+  		'C425',
+  		'C441',
+  		'C500',
+  		'C501',
+  		'C510',
+  		'C525',
+  		'C526',
+  		'C550',
+  		'C551',
+  		'C560',
+  		'C56X',
+  		'C650',
+  		'C680',
+  		'C72R',
+  		'C750',
+  		'C77R',
+  		'C82R',
+  		'COL3', //Lancair (Source: http://www.icao.int/publications/DOC8643/Pages/ManufacturersName.aspx)
+  		'COL4',
+  		'LNC2',
+  		'LNCE',
+  		'EVOT',
+  		'LNT4',
+  		'TGRS',
+  		'LEG2',
+  		'SNGY',
+  		'EVOP',
+  		'LNP4',
+   		'LJ23', //Learjet
+  		'LJ24',
+  		'LJ25',
+  		'LJ31',
+  		'LJ35',
+  		'LJ40',
+  		'LJ45',
+  		'LJ55',
+  		'LJ60',
+  		'AEST', //Piper
+  		'CRUZ',
+  		'J2',
+  		'J3',
+  		'J4',
+  		'J5',
+  		'P28A',
+  		'P28B',
+  		'P28R',
+  		'P28S',
+  		'P28T',
+  		'P28U',
+  		'P32R',
+  		'P32T',
+  		'P46T',
+  		'PA11',
+  		'PA12',
+  		'PA14',
+  		'PA15',
+  		'PA16',
+  		'PA17',
+  		'PA18',
+  		'PA20',
+  		'PA22',
+  		'PA23',
+  		'PA24',
+  		'PA25',
+  		'PA27',
+  		'PA30',
+  		'PA31',
+  		'PA32',
+  		'PA34',
+  		'PA36',
+  		'PA38',
+  		'PA44',
+  		'PA46',
+  		'PA47',
+  		'PAT4',
+  		'PAY1',
+  		'PAY2',
+  		'PAY3',
+  		'PAY4',
+  		'PILL',
+  		'S108',
```

Comment:
This comma is obsolete, but I'll remove it when implementing the functionality for #27.